graylog2-server/src/main/java/org/graylog/plugins/views/search/errors/SearchTypeErrorParser.java

```diff
@@ -43,6 +48,15 @@ public static SearchTypeError parse(Query query, String searchTypeId, Elasticsea
         return new SearchTypeError(query, searchTypeId, ex);
     }
 
+    private static boolean isSearchTypeAbortedError(ElasticsearchException ex) {
+        return ex != null &&
+                (
+                        ex.getMessage().contains("type=task_cancelled_exception")
+                                || (ex.getCause() != null && ex.getCause() instanceof TimeoutException)
```

Comment:
Is this always correct, that a TimeoutException as a cause means it got aborted? Could the timeout not point to other problems in the subsystem?

Reply:
Maybe `aborted` is a bad name and I should name it differently? 
As for me it is not that important if it is a cancel or timeout, it is important it did not finish, which I called aborted...